collections/_posts/2022-07-07-C2FTrans-Segmentation.md

```diff
@@ -131,7 +131,7 @@ The authors mention the comparison between the SOTA methods but it lacks the com
 # Conclusions
 The introduction of C2F transformer in medical image segmentation is interesting, especially the Cross-scale Global Transformer. However, the paper is not well written as some important details are missing, e.g. the generation and combination of $$ Q $$, $$ K $$, and $$ V $$ of different dimension in the CGT. Moreover, there are some contradictions between some of the steps described in their paper and their GitHub code. 
 
-Their GitHub repository is hard to use (incorrect _requirements.txt_ to setup a working virtual environment). I have tested their algorithm on the Camus dataset and the results were worse than those given by nnUNet, which made me doubt the correctness of their results.
+Their GitHub repository is hard to use (incorrect _requirements.txt_ to setup a working virtual environment, barely commented codes, etc.). I have tested their algorithm on the Camus dataset and the results were worse than those given by nnUNet, which made me doubt the correctness of their results.
```

Comment:
Take care, the site is public, I would suggest to smooth a little bit the last sentence by removing the part: "which made me doubt the correctness of their results.". I guess the message remains the same

Reply:
Thanks Olivier for the catch!